pom.xml

```diff
@@ -277,9 +276,9 @@
 			<version>8.1.0</version>
 		</dependency>
 		<dependency>
-			<groupId>org.rxtx</groupId>
-			<artifactId>rxtx</artifactId>
-			<version>2.1.7</version>
+		  <groupId>com.fazecast</groupId>
+		  <artifactId>jSerialComm</artifactId>
+		  <version>2.10.4</version>
```

Comment:
This dependency group is not indented as much as those around it.   It looks like 2 spaces where the others are indented 4 spaces.